articles/understanding-min-heap-vs-max-heap/index.md

```diff
@@ -0,0 +1,520 @@
+A heap is a tree-based data structure that allows access to the minimum and maximum element in the tree in constant time. They reduce the normal run time with polynomial order. There are two types of heaps: Min-heap and Max-heap. Min-heap is used for accessing the minimum element in the heap whereas the Max-heap is used for accessing the maximum element in the heap.
```

Comment:
`They reduce the normal run time with polynomial order`. Can you explain this better? What do you mean by normal runtime?

Reply:
Solved it by explaining constant time taken.